examples/design-system/src/tokenami.css

```diff
@@ -23,19 +23,25 @@ body {
   .theme-light {
     --_grid: .25rem;
     --anim_wiggle: wiggle 1s ease-in-out infinite;
+    --border_thin: 1px solid var(--color_slate-700);
+    --color_slate-700: #334155;
     --color_primary: #f1f5f9;
     --color_secondary: #334155;
-    --color_slate-700: #334155;
     --font_sans: sans-serif;
     --radii_rounded: 10px;
   }
 
+  [style] {
+    --border_thin: 1px solid var(--color_slate-700);
+  }
```

Comment:
we don't want this to happen for vars than aren't custom properties. i have another branch in progress to solve this.